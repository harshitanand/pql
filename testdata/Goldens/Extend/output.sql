WITH "__subquery0" AS (SELECT "State" AS "State", "EventType" AS "EventType", "DamageProperty" AS "DamageProperty" FROM "StormEvents")
SELECT *, 1 AS "foo" FROM "__subquery0";
