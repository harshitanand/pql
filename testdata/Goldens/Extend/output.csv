State,EventType,DamageProperty, foo
ATLANTIC SOUTH,Waterspout,0,1
FLORIDA,Heavy Rain,0,1
FLORIDA,Tornado,6200000,1
GEORGIA,Thunderstorm Wind,2000,1
MISSISSIPPI,Thunderstorm Wind,20000,1
